interface.go

```diff
@@ -29,6 +28,7 @@ type Manager interface {
 	List(context.Context, Filter) ([]Process, error)
 	Group(context.Context, string) ([]Process, error)
 	Get(context.Context, string) (Process, error)
+	Limit(context.Context, interface{}) error
```

Comment:
as per our conversation in person, I think it's best to hold off on this now, and then revist this method when we have a use case